Furyfile.yml

```diff
@@ -1,15 +1,15 @@
-# Copyright (c) 2022 SIGHUP s.r.l All rights reserved.
+# Copyright (c) 2023 SIGHUP s.r.l All rights reserved.
 # Use of this source code is governed by a BSD-style
 # license that can be found in the LICENSE file.
 
 versions:
-  networking: v1.10.0
+  networking: v1.12.1
   monitoring: v2.0.1
```

Comment:
before releasing we should try to backport the patch to the [issue with alert manager's configuration](https://github.com/sighupio/fury-kubernetes-monitoring/issues/132)

Reply:
Ok